.github/workflows/pr-helm-upgrade-testing.yaml

```
@@ -0,0 +1,330 @@
+name: "Helm upgrade test"
+description: |
```

actionlint (1.7.8)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)